### DISCUSSION TITLE
v0.20.0

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#3146] [Versions & Releases]
## What's Changed
* Run `make style` & re-enable it in CI by @akx in https://github.com/coqui-ai/TTS/pull/3127
* XTTS v2.0 by @Edresson in https://github.com/coqui-ai/TTS/pull/3137

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.19.1...v0.20.0

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.20.0'>v0.20.0</a>.</em>

**Comment by jianchang512** [comment]
when will support Chinese language?

**Reply by kdavis-coqui** [reply]
It's supported Chinese since the middle of September.
